### PR TITLE
Make all fields of CFD private

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: daemon/src/model.rs [hunk 1]
```diff
@@ -390,7 +390,7 @@ impl From<Decimal> for Percent {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, sqlx::Type)]
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, sqlx::Type)]
 pub enum TradingPair {
```
Comment (klochowicz) [inline daemon/src/model.rs:394]: do you need to implement `Clone` anymore here?

**Review by klochowicz — APPROVED** [review]
good work!
I like the new getter `state_mut()` as it makes it a bit clearer that we're going to change the state here. (and it will be easier to grep for places where we actually change the state :) ).

File: daemon/src/model.rs (hunk 1 above)
Comment (thomaseizinger) [reply daemon/src/model.rs:394]: You need `Clone` to derive `Copy`!

**Comment by thomaseizinger** [comment]
> I like the new getter `state_mut()` as it makes it a bit clearer that we're going to change the state here. (and it will be easier to grep for places where we actually change the state :) ).

Well, it is going away with event-sourcing again :joy:

File: daemon/src/model.rs (hunk 1 above)
Comment (klochowicz) [reply daemon/src/model.rs:394]: ah, indeed!

**Pull request merged**